src/gtk/webview_webkit2.cpp

```diff
@@ -252,15 +252,18 @@ wxgtk_webview_webkit_load_failed(WebKitWebView *,
     {
         switch (error->code)
         {
+#if SOUP_MAJOR_VERSION < 3
```

Comment:
Does this mean that we don't get `CANCELLED` error at all any longer or do we still get it in some other way? Looking at https://libsoup.org/libsoup-3.0/migrating-from-libsoup-2.html#status-codes-no-longer-used-for-internal-errors it seems like we should examine `GError` for it but I haven't looked at this at all and just wanted to ask if you already had.